src/DynamoCore/Graph/Workspaces/PackageDependencyInfo.cs

```diff
@@ -73,10 +73,54 @@ public override string ToString()
         }
     }
 
+    internal enum ReferenceType{
```

Comment:
Seems this enum is specific to `PackageDependencyInfo` object and we may need to do comparison later from other places, I suggest we move it from this name space to that class as a static enum.

Reply:
it's a top level enum inside the namespace `Dynamo.Graph.Workspaces`, not nested under the `PackageDependencyInfo` class -  I can move it to interfaces if you think that makes more sense?

Reply:
Well that's what I mean.. I think the package dependency view extension later will need to access this enum and do comparison with the referenceType so it would be good for us to make it static